src/templates/content.js

```diff
@@ -8,6 +8,7 @@ const ContentPage = ({ data }) => {
     <Layout
       title={page.title}
       navigation={page.navigationGroup ? page.navigationGroup.pages : false}
+      narrow="true"
```

Comment:
Props that are truthy should just be empty, so just change `narrow="true"` to `narrow`

```javascript
<Layout [stuff] narrow>
```

Reply:
Thanks, makes sense. Updated.

Reply:
🙌